tools/tsdb/bloom-tester/lib.go

```diff
@@ -378,7 +378,6 @@ func analyze(metrics *Metrics, sampler Sampler, indexShipper indexshipper.IndexS
 
 											metrics.sbfCreationTime.WithLabelValues(experiment.name).Add(float64(endTime - startTime))
 											metrics.sbfsCreated.WithLabelValues(experiment.name).Inc()
-											metrics.chunkSize.Observe(float64(chunkTotalUncompressedSize))
```

Comment:
Since this is not experiment-specific, and is being captured in the PopulateSeriesWithBloom function, remove it from the lib.go file.